ap_api/resources/get-quote.api.mdx

```diff
@@ -0,0 +1,47 @@
+---
+id: get-quote
+title: "Fetch a single quote provided to a client application."
+description: "Only relevant for SEP-38 (used by SEP-31). Fetches the quote that ascreated by a client application using SEP-38's `POST /quote` endpoint."
+sidebar_label: "Fetch a single quote provided to a client application."
+hide_title: true
+hide_table_of_contents: true
+api: {"tags":["Quotes"],"operationId":"getQuote","description":"Only relevant for SEP-38 (used by SEP-31). Fetches the quote that ascreated by a client application using SEP-38's `POST /quote` endpoint.","parameters":[{"in":"path","name":"id","description":"The ID returned as a result of creating the quote via SEP-38's `POST /quote` endpoint.","schema":{"type":"string"},"required":true}],"responses":{"200":{"description":"Success","content":{"application/json":{"schema":{"type":"object","properties":{"id":{"type":"string"},"sell_amount":{"type":"string"},"sell_asset":{"type":"string"},"buy_amount":{"type":"string"},"buy_asset":{"type":"string"},"expires_at":{"type":"string","format":"date-time"},"price":{"type":"string"},"total_price":{"type":"string"},"creator":{"type":"object","description":"StellarId's are objects that identify end-users and SEP-31 Sending Anchors, but not SEP-31 Receiving Anchors.\n\nFor a SEP-12 customer, the `id` field should be sufficient to fully identify the customer in the Anchor Backend.\n\nFor a SEP-31 Sending Anchor, the `account` field should be used.\n","properties":{"id":{"type":"string","description":"The `id` of the customer registered through SEP-12."},"account":{"type":"string","description":"Either the Stellar account or Muxed account address of the on-chain entity."}}},"transaction_id":{"type":"string"},"created_at":{"type":"string","format":"date-time"}}}}}},"404":{"description":"Not Found.","content":{"application/json":{"schema":{"type":"object","properties":{"error":{"type":"string"},"id":{"type":"string"}},"required":["error"]}}}}},"method":"get","path":"/exchange/quotes/{id}","info":{"version":"2.0.0","title":"Platform API"},"postman":{"name":"Fetch a single quote provided to a client application.","description":{"content":"Only relevant for SEP-38 (used by SEP-31). Fetches the quote that ascreated by a client application using SEP-38's `POST /quote` endpoint.","type":"text/plain"},"url":{"path":["exchange","quotes",":id"],"host":["{{baseUrl}}"],"query":[],"variable":[{"disabled":false,"description":{"content":"(Required) The ID returned as a result of creating the quote via SEP-38's `POST /quote` endpoint.","type":"text/plain"},"type":"any","value":"","key":"id"}]},"header":[{"key":"Accept","value":"application/json"}],"method":"GET"}}
```

Comment:
This is not ideal, is this how we have to do it? Aren't we duplicating data between this and the yaml files?

Reply:
What do you mean? It's auto-generated